tests/test_views.py

```diff
@@ -7,13 +7,20 @@
 import requests
 from rest_framework.test import APIClient
 
+from s3_file_field import _multipart
+
 from .fuzzy import URL_RE, UUID_RE, Re
 
 
 def mb(bytes_size: int) -> int:
     return bytes_size * 2 ** 20
 
 
+@pytest.fixture
+def mock_part_size(monkeypatch):
```

Comment:
This fixture needs to be used much more broadly. A number of other unit tests are designed around a 5MB size, and are invalidated with a larger size, even if they don't explicitly fail.

Reply:
Should it just be marked as an autouse fixture?